### ISSUE TITLE
preferNoSetFee not working

### ISSUE THREAD

**Issue by ap-justin** [chainapsis/keplr-extension#201]
Hi! I would really appreciate some help with this 
https://stackblitz.com/edit/react-ts-pezzyd?file=Keplr2.tsx

I am trying to send a transaction with custom fee and set the interaction with 
```
window.keplr.defaultOptions = {
      sign: { preferNoSetFee: true },
};
  
```
but keplr still suggest fees and doesn't get overriden by the custom fee 

use case: 
staking currency : `coinA`
fee currencies: `[coinA, coinB]`

I want to use `coinB` as the fee denom but kelpr still suggest `coinA`

**Comment by Thunnini** [comment]
I can't run the demo because I don't have an account on the your testnet

**Comment by ap-justin** [comment]
@Thunnini, I've tried to put in codesandbox instead
https://codesandbox.io/s/silly-surf-o795u, and used public tesnet rpc endpoint `https://rpc.testnet.cosmos.network:443`
I will really appreciate some feedbacks

**Comment by Thunnini** [comment]
@ap-justin Can you send some test token to me? cosmos1vv6hruquzpty4xpks9znkw8gys5x4nsnqw9f4k

**Comment by ap-justin** [comment]
@Thunnini sent your ser some uphotons!

**Comment by Thunnini** [comment]
Oops. There was an issue where options were not being passed properly in signDirect due to a simple mistake. I will fix it in the next version.

**Comment by ap-justin** [comment]
@Thunnini big thanks ser for looking! I just realized I misunderstood you about having no account in testnet x_x

**Comment by Thunnini** [comment]
This should be resolved in v0.9.9
